### PR TITLE
[Bigtable] CreateTableRequest equals and hashCode with unit test

### PR DESCRIPTION
**Pull request by rahulKQL**
More information: https://github.com/googleapis/google-cloud-java/pull/5066#discussion_r281310882

- Fixed `CreateTableRequest#equals` & `hashCode`
- Added unit test for equals, hashCode & toProto.

### CODE REVIEW

**Comment by rahulKQL** [comment]
@igorbernstein2 Please have a look

File: google-cloud-clients/google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/models/CreateTableRequestTest.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                                    .build())
+                            .build()))
+            .setParent(NameUtil.formatInstanceName(PROJECT_ID, INSTANCE_ID))
+            .addInitialSplits(
+                com.google.bigtable.admin.v2.CreateTableRequest.Split.newBuilder().setKey(splitKey))
+            .addInitialSplits(
+                com.google.bigtable.admin.v2.CreateTableRequest.Split.newBuilder()
+                    .setKey(secondSplitKey))
+            .build();
+    Truth.assertThat(request.toProto(PROJECT_ID, INSTANCE_ID)).isEqualTo(requestProto);
```
Comment (igorbernstein2) [inline google-cloud-clients/google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/models/CreateTableRequestTest.java]: Please use a static import for assertThat

File: google-cloud-clients/google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/models/CreateTableRequestTest.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            .setTableId(TABLE_ID)
+            .setTable(
+                Table.newBuilder()
+                    .putColumnFamilies("family-id", ColumnFamily.getDefaultInstance())
+                    .putColumnFamilies(
+                        "another-family",
+                        ColumnFamily.newBuilder()
+                            .setGcRule(
+                                GcRule.newBuilder()
+                                    .setMaxAge(Duration.newBuilder().setSeconds(100 * 60 * 60))
```
Comment (igorbernstein2) [inline google-cloud-clients/google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/models/CreateTableRequestTest.java]: When there is a conflict between internal proto class names (com.google.protobuf.Duration) and external class names (org.threeten.bp.Duration) please use fully qualified names for the proto names and short names for the external names

File: google-cloud-clients/google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/models/CreateTableRequestTest.java (hunk 1 above)
Comment (rahulKQL) [reply google-cloud-clients/google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/models/CreateTableRequestTest.java]: Imported `assertThat` statically now.

File: google-cloud-clients/google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/models/CreateTableRequestTest.java (hunk 2 above)
Comment (rahulKQL) [reply google-cloud-clients/google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/models/CreateTableRequestTest.java]: Thanks for the explanation, will be following this pattern from now on. Have updated with the same.

**Comment by rahulKQL** [comment]
@igorbernstein2 This PR is ready for another round of review, PTAL.

**Pull request merged**